src/libs/actions/Session/index.js

```diff
@@ -22,13 +22,23 @@ import subscribeToReportCommentPushNotifications from '../../Notification/PushNo
 import ROUTES from '../../../ROUTES';
 import * as ErrorUtils from '../../ErrorUtils';
 import * as ReportUtils from '../../ReportUtils';
-import * as Report from '../Report';
 import {hideContextMenu} from '../../../pages/home/report/ContextMenu/ReportActionContextMenu';
 
 let authTokenType = '';
+let sessionAuthToken = null;
```

Comment:
Since we already have `authTokenType` without _session_ prefix, let's keep it consistently.  
```suggestion
let authToken = null;
```

Reply:
yeah but it collides with a var used in a scope of a function somewhere in the file. That's why I gave it the 'session' prefix

Reply:
Makes sense. So, there is an option to rename `authTokenType` to `sessionAuthTokenType`, or just leave as it is.